_posts/2020-06-05-jason.md

```diff
@@ -0,0 +1,11 @@
+---
+title: This summer I'm looking forward to...
+author: Jason
+date: 2020-06-05
+---
+
+This summer I'm looking forward to running!
+
+![Image of GitHub Cat](https://github.githubassets.com/images/modules/open_graph/github-octocat.png)
+
+Comment your reply here.
```

Comment:
Nice work here Jason. I need to start running more. Got any tips?
Also, not sure how the github octocat relates to running - do you have 8 legs? I'm not sure if that would be an unfair advantage or just make things more difficult. (ha ha)